Setup/UpgradeSchema.php

```diff
@@ -59,8 +59,8 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
             $this->updateSchemaVersion204($setup);
         }
 
-        if (version_compare($context->getVersion(), '2.0.7', '<')) {
-            $this->updateSchemaVersion207($setup);
+        if (version_compare($context->getVersion(), '2.1.0', '<')) {
+            $this->updateSchemaVersion210($setup);
```

Comment:
In case version 2.0.7 is released, this will cause problems to the ones upgrading from 2.0.7 to 2.1.0. Maybe is better idea to release 2.0.7 instead of 2.1.0 ?